src/components/FormPageComponents/FormPageComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import form from "../../images/form.jpg";
+import { makeStyles } from "@material-ui/core";
+// import ResponsiveAppBar from "./ResponsiveMenuAppBar";
+import NewAppBar from "./NewAppBar";
+const useStyles = makeStyles({
+  imgStyle: {
+    width: "100%",
+    height: "40vh",
+  },
+});
+
+function FormPageComponent() {
+  const classes = useStyles();
+  return (
+    <>
```

Comment:
I think it will be better if you create two separate appbars, one for mobile and one for normal screen. You have display dependent from screen in many places, it will be better if you will have that once.